site/content/docs/5.2/utilities/colors.md

```diff
@@ -14,10 +14,16 @@ Colorize text with color utilities. If you want to colorize links, you can use t
 {{< colors.inline >}}
```

Comment:
In dark mode, some examples should have a white background so that the texts remain visible (like we do in light mode by adding a black background behind some texts). Worst case being `.text-dark` which is invisible.